runtime/vm/JVMImage.cpp

```diff
@@ -142,6 +261,34 @@ JVMImage::freeSubAllocatedMemory(void* address)
 	omrthread_monitor_exit(_jvmImageMonitor);
 }
 
+void
+JVMImage::registerEntryInTable(ImageTableHeader *table, UDATA entry)
+{
+	Trc_VM_RegisterInTable_Entry(table, table->currentSize, *(WSRP_GET(table->tableTail, UDATA*)), entry);
+
+	// table is not large enough and needs to be reallocated
+	if (table->currentSize >= table->tableSize) {
+		reallocateTable(table, table->tableSize * 2); //TODO: Introduce error handling for table reallocation
+	}
+
+	UDATA *head = WSRP_GET(table->tableHead, UDATA*);
+	UDATA *tail = WSRP_GET(table->tableTail, UDATA*);
+
+	// initial state of every table
+	if (table->currentSize == 0) {
+		*(head) = entry;
+	}
+	else {
```

Comment:
Nitpick: bracing should be `} else {`

have you guys seen the coding guidelines? 
This is the OMR one, https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md its the same as OpenJ9

Reply:
removed else statement (see above). I dont think we followed these guidelines. Will put a task to do a complete fix using these guidelines